docs/conf.py

```diff
@@ -0,0 +1,115 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'rmf_ros2'
+copyright = '2021, Open Source Robotics Corporation'
+author = 'Open Source Robotics Corporation'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0.0'
```

Comment:
Should we consider making this a `.in` template file so that we can have cmake automatically generate it from the version number that we declare in cmake? Otherwise we'll need to remember to update this field with each version change.